spec/order_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'order'
+require 'menu'
+
+describe Order do
+  subject(:order) { described_class.new(menu) }
+  let(:menu) { double(:menu, print: printed_menu) }
+  let(:printed_menu) { "Pasta - £7, Pizza - £9, Canlone - £9" }
+  let(:order_list) { {Pizza: 3} }
+
+  it "should initialze with an empty order list array" do
+    expect(order.order_list).to be_instance_of(Hash)
```

Comment:
I think it's usually better to do one expectation per description - so separate this expect and the next one into separate it blocks